appsec/src/extension/commands_helpers.h

```diff
@@ -36,10 +36,11 @@ dd_result dd_command_proc_resp_verd_span_data(mpack_node_t root,
     void *unspecnull ctx /* actually struct req_info* nonnull */);
 
 /* Common helpers */
+void dd_command_process_settings(mpack_node_t root);
 void dd_command_process_meta(mpack_node_t root, zend_object *nonnull span);
 bool dd_command_process_metrics(mpack_node_t root, zend_object *nonnull span);
 bool dd_command_process_telemetry_metrics(mpack_node_t root);
 dd_result dd_command_process_config_features(
     mpack_node_t root, ATTR_UNUSED void *nullable ctx);
 dd_result dd_command_process_config_features_unexpected(
-    mpack_node_t root, ATTR_UNUSED void *nullable ctx);
+    mpack_node_t root, ATTR_UNUSED void *nullable ctx);
```

Comment:
Missing empty line

Reply:
Actyally, you removed it 😅

Reply:
I fixed the multiple missing empty lines, my bad :sweat_smile: